lib/features/modeling/BpmnFactory.js

```diff
@@ -94,4 +94,14 @@ BpmnFactory.prototype.createDiPlane = function(semantic) {
   });
 };
 
+BpmnFactory.prototype.createEventDefinition = function(type, parent) {
+  var element = this.create(type, {
```

Comment:
This won't work, as an event may have more than one event definition; in that case, the ids would collide.

Instead, let `BpmnFactory` assign a random id.